browser/autoplay/autoplay_permission_context_unittest.cc

```diff
@@ -44,7 +44,7 @@ class AutoplayPermissionContextTest : public AutoplayPermissionContext {
   void NotifyPermissionSet(const PermissionRequestID& id,
                            const GURL& requesting_origin,
                            const GURL& embedder_origin,
-                           const BrowserPermissionCallback& callback,
+                           const BrowserPermissionCallback callback,
```

Comment:
Is that `const` meant to be there? It's not doing any harm, just a little inconsistent with the other changes in this commit.

Reply:
Fixed.